activiti-api/activiti-process-model-api-impl/src/main/java/org/conf/activiti/runtime/ProcessModelAutoConfiguration.java

```diff
@@ -81,39 +80,39 @@ public JavaType resolveAbstractType(DeserializationConfig config,
                             ProcessDefinitionImpl.class);
         resolver.addMapping(SequenceFlow.class,
                             SequenceFlowImpl.class);
-
         resolver.addMapping(IntegrationContext.class,
                             IntegrationContextImpl.class);
+        resolver.addMapping(Result.class,
```

Comment:
@erdemedeiros I've tried that but it doesn't work. Because we have Result<T> I needed to create more specific classes for Jackson to figure out which result type it is. So even if we want to do that, we can't. 
The mapping as it is works, because the Mixin check the Result type to figure out if it is a Task or a ProcessInstance result.

Reply:
No sure that I completely got your point. My point was this mapping is probably unnecessary because as you mentioned it's going to use the MixIn mapping (line 93). I'll try to remove line 85 and see it still works.